From: Ole Streicher <olebole@debian.org>
Date: Thu, 1 Dec 2016 21:18:56 +0100
Subject: Add SONAME to libastrometry.so

---
 solver/Makefile | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/solver/Makefile b/solver/Makefile
index 5e1bd09..2b9f0ac 100644
--- a/solver/Makefile
+++ b/solver/Makefile
@@ -128,7 +128,8 @@ $(ENGINE_LIB): $(ENGINE_OBJS)
 	$(RANLIB) $@
 
 $(ENGINE_SO): $(ENGINE_OBJS) $(SLIB)
-	$(CC) $(LDFLAGS) $(SHAREDLIBFLAGS) -o $@ $^ $(LDLIBS)
+	$(CC) $(LDFLAGS) $(SHAREDLIBFLAGS) -Wl,-soname,$@.0 -o $@.0 $^ $(LDLIBS)
+	ln -s  $@.0 $@
 
 # old and miscellaneous executables that aren't part of the pipeline.
 OLDEXECS := checkquads
@@ -199,8 +200,8 @@ install: $(INSTALL_EXECS) $(INSTALL_LIB)
 	done
 	$(MKDIR) '$(LIB_INSTALL_DIR)'
 	@for x in $(INSTALL_LIB); do \
-		echo $(CP) $$x '$(LIB_INSTALL_DIR)'; \
-		$(CP) $$x '$(LIB_INSTALL_DIR)'; \
+		echo $(CP) -d $$x* '$(LIB_INSTALL_DIR)'; \
+		$(CP) -d $$x* '$(LIB_INSTALL_DIR)'; \
 	done
 	$(MKDIR) '$(PY_INSTALL_DIR)'
 	@for x in $(PYTHON_INSTALL); do \
