--- a/docs/conf.py	2020-11-25 15:28:48.000000000 +0800
+++ b/docs/conf.py	2021-01-09 18:25:49.501862265 +0800
@@ -114,7 +114,13 @@
 # built documents.
 
 # The full version, including alpha/beta/rc tags.
-release = get_distribution(project).version
+try:
+    release = get_distribution(project).version
+except Exception:
+    # assume local build
+    sys.path.append('..')
+    from astropy import __version__ as release
+
 # The short X.Y version.
 version = '.'.join(release.split('.')[:2])
 
