--- a/astropy_helpers/commands/build_sphinx.py	2018-06-02 03:58:03.000000000 +0800
+++ b/astropy_helpers/commands/build_sphinx.py	2019-12-21 23:00:08.630476872 +0800
@@ -119,7 +119,7 @@
         build_cmd = self.get_finalized_command('build')
         build_cmd_path = os.path.abspath(build_cmd.build_lib)
 
-        ah_importer = pkgutil.get_importer('astropy_helpers')
+        ah_importer = pkgutil.get_importer('/usr/lib/python3.6/site-packages/astropy_helpers')
         ah_path = os.path.abspath(ah_importer.path)
 
         # Now generate the source for and spawn a new process that runs the
